src/Compiler/Facilities/DiagnosticsLogger.fs

```diff
@@ -412,6 +412,10 @@ type internal DiagnosticsThreadStatics =
 [<AutoOpen>]
 module DiagnosticsLoggerExtensions =
 
+    let PutStackTraceToErrors =
+        (Environment.GetEnvironmentVariable("FSC_PutStackTrace_To_Errors") = "1")
+        || false
```

Comment:
Why's this?

Reply:
Thanks for spotting this, this should go away alltogether (is a debugging leftover so that CI prints full stacktraces when we produce an error. Especially painful for internal errors when we do not have stacktraces)

Reply:
I could keep it of course, but it would mess up with test outputs anyway (since the stacktrace would be there)

Reply:
Ooh, that's handy. Can we leave it please?